src/resources/i18n/translations/en.json

```diff
@@ -148,8 +148,8 @@
 	"ARIA HINT - show more options for this": "Show more options for this",
 	"ARIA HINT - show more player screen options": "Show more player screen options",
 	"ARIA HINT - show the chapters for this episode": "Show the chapters for this episode",
-	"ARIA HINT - sign up for a premium membership": "Sign up for a premium membership",
-	"ARIA HINT - sign up for your premium membership": "Sign up for your premium membership",
+	"ARIA HINT - sign up for a premium membership": "Sign up for a Premium membership",
+	"ARIA HINT - sign up for your premium membership": "Sign up for your Premium membership",
```

Comment:
```suggestion
	"ARIA HINT - sign up for your premium membership": "Sign up for Podverse Premium",
```

Reply:
These "ARIA HINT" translations are only read by screen readers. In this case it is meant as a hint (if you're not familiar with how accessibility hints work lmk) to the user that the "Sign Up" button will allow you to "Sign up for a Podverse Premium membership". Can you change it to that?